.github/workflows/release-beta.yaml

```
@@ -0,0 +1,39 @@
+      pull-requests: read
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: latest
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)